src/metpy/calc/thermo.py

```diff
@@ -1052,12 +1052,6 @@ def parcel_profile_with_lcl(pressure, temperature, dewpoint):
     >>> # computer parcel temperature
     >>> Td = dewpoint_from_relative_humidity(T, rh)
     >>> p_wLCL, T_wLCL, Td_wLCL, prof_wLCL = parcel_profile_with_lcl(p, T, Td)
-    >>> print(f'Shape of original pressure array: {p.shape}')
```

Comment:
I was attempting to show that the function adds a new level in with printing out the shape the array. I don't think it is necessary, so fine to remove, but just wanted to document about why I had them in there in the first place when adding the examples to the doc strings.

Reply:
Sure. I think it's definitely good to document, but the example, which focuses on usage, probably isn't quite the right place (since it makes the example so verbose). Regardless, wasn't the source of the problem, but felt like a good cleanup.